data/korath/korath missions.txt

```diff
@@ -257,3 +257,331 @@ mission "Small Scale Delivery"
 	on complete
 		payment 1000
 		dialog `You find a Korath who seems to have been waiting for you in the spaceport. They greet you by showing their palms, then outstretch their hands as if waiting to receive the package. You hand off the small delivery and they hand you a small payment of <payment>.`
+
+
+substitutions
+	"<EDC-ship>" "Esketari"
+
+mission "Efreti: Distress Call 1"
+	minor
+	landing
+	name "Distress call from the <EDC-ship>"
+	description "The Quarg have asked you investigate a distress call from the Efreti ship <EDC-ship> which has crashed on <destination> with <bunks> survivors."
+	source "Karek Fornati"
+	destination "Seleptra Nak"
+	passengers 40
+	blocked "A Quarg ship broadcasts that they need a ship with at least <bunks> bunks for an emergency evacuation."
```

Comment:
Can't the Quarg send their own ships to help? If they're right there, then they should be able to help some.